tensorflow/core/BUILD

```diff
@@ -1762,6 +1762,9 @@ tf_cc_tests(
         "graph/graph_test.cc",
         "graph/node_builder_test.cc",
         "graph/optimizer_cse_test.cc",
+# INTEL_MKL start
```

Comment:
Oh, got it.  In that case, you can try putting this in a standalone rule like the "cuda_libdevice_path_test" test below.